lib/bolt/pal.rb

```diff
@@ -67,6 +68,10 @@ def self.configure_logging
       Puppet.settings[:log_level] = 'debug'
     end
 
+    def self.with_trusted_external_command(trusted_external_command)
```

Comment:
Should we do this in `with_puppet_settings` instead? The purpose of that method is to manage the lifecycle of our Puppet settings.